compiler/rustc_error_codes/src/error_codes/E0536.md

```diff
@@ -3,22 +3,20 @@ The `not` cfg-predicate was malformed.
 Erroneous code example:
 
 ```compile_fail,E0536
-#[cfg(not())] // error: expected 1 cfg-pattern
-pub fn something() {}
-
-pub fn main() {}
+pub fn main() {
+    if cfg!(not()) { }
+}
```

Comment:
This error code will not be needed in the future (as wel as many others), but needed a new example for now